revscoring/languages/tests/test_bosnian.py

```diff
@@ -0,0 +1,151 @@
+import pickle
+
+from nose.tools import eq_
+
+from .. import bosnian
+from ...datasources import revision_oriented
+from ...dependencies import solve
+from .util import compare_extraction
+
+BAD = [
+    "balija",
+    "debil",
+    "debili",
+    "debilu",
+    "drolja",
+    "drolje",
+    "droljetine",
+    "droljo",
+    "fašisti",
+    "fuck",
+    "govna",
+    "govno",
+    "iliriski",
+    "jebanje",
+    "jebe",
+    "jebem",
+    "jebem",
+    "jebi",
+    "jebiga",
+    "jebite",
+    "jebo",
+    "kreten",
+    "kretenčina",
+    "kretenčuga",
+    "kreteni",
+    "kretenu",
+    "kučka",
+    "kuja",
+    "kuje",
+    "kurac",
+    "kurce",
+    "kurcina",
+    "kurčina",
+    "kurva",
+    "lezbac",
+    "lezbać",
+    "maloglavi",
+    "materinu",
+    "mrš",
+    "peder",
+    "pederi",
+    "pederima",
+    "pederski",
+    "pederu",
+    "picka",
+    "pička",
+    "picke",
+    "pičke",
+    "picko",
+    "pičko",
+    "pizda",
+    "pizdo",
+    "pizdu",
+    "puškomet",
+    "shit",
+    "sranje",
+    "šupak"
+]
+
+INFORMAL = [
+    "boriću",
+    "bubaj",
+    "drzava",
+    "glup",
+    "haha",
+    "hahaha",
+    "hahahaha",
+    "hahahahaha",
+    "hihi",
+    "hihihi",
+    "istorija",
+    "lmao",
+    "lol",
+    "nesto",
+    "neznam",
+    "nista",
+    "opština",
+    "opštini",
+    "orgaizovanje",
+    "pokusava",
+    "pokušavajuci",
+    "povijest",
+    "povješničari",
+    "pregledaču",
+    "šireči",
+    "takođerr",
+    "ucvršćivanja",
+    "vecina",
+    "zivjeli",
+    "zivjelo"
+]
+
+OTHER = [
+    """
+    On je sjajni, srebreno-sivi četverovalentni prelazni metal. U hemijskom
+    smislu, dosta je sličan cirkoniju a može se naći i u mineralima
+    cirkonija. Njegovo postojanje je predvidio Mendeljejev već 1869. godine,
+    ali sve do 1923. nije identificiran kao element. Bio je pretposljednji
+    element sa stabilnim izotopima koji je otkriven (renij je identificiran
+    dvije godine kasnije).
+    """
+]
+
+r_text = revision_oriented.revision.text
+
+
+def test_badwords():
+    compare_extraction(bosnian.badwords.revision.datasources.matches,
```

Comment:
I see that we use this pattern for every language, so no the fault of this revision, but I'm personally not a fan of this type of test.  We can assert that the list can be unpickled, that it's longer than 10 or something, but keeping two copies of the same list in sync seems counterproductive.

Reply:
@adamwight, note that these lists diverge when someone has time to take advantage of regular expressions in the primary list.  See enwiki.py for example.  Also note that this test confirms that we are able to extract badwords when there is text surrounding it.  This is extra important when we have regular expressions involved that can match whitespace and other word breaking chars.